lib/ui/views/home/home_viewmodel.dart

```diff
@@ -203,10 +204,10 @@ class HomeViewModel extends BaseViewModel {
                         iOSAppId: 'com.notes.ounotes');
                   }
                   // This allows to mimic the behavior of the default "Rate" button. See "Advanced > Broadcasting events" for more information :
-                  await rateMyApp
-                      .callEvent(RateMyAppEventType.rateButtonPressed);
-                  Navigator.pop<RateMyAppDialogButton>(
-                      context, RateMyAppDialogButton.rate);
+                  // await rateMyApp
```

Comment:
### Best practices being violated
Please see to it that you don't keep outdated code around, it tends to clutter the repo and in turn increasing my blood pressure.
Thank you.